### PR TITLE
feat: added featured wallets this week

### LINKED ISSUE #699: New highlighted wallet accounts on the `top` page.
Just like the featured org and planters, we need a section of `Featured wallet this week` and the UI could be the same, but list 10 newest wallets with logo (we will replace it with real featured list in the future) 
image

---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.
- More issues in this project: https://github.com/orgs/Greenstand/projects/26

### PR DESCRIPTION
**Pull request by jeremydthomas**
# Description

[comment]: # 'Added wallets of the week section to TOP page'

Fixes #699 

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [x ] New feature (non-breaking change which adds functionality)

## Screenshots

image

# How Has This Been Tested?

visual

# Checklist:

- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings

### CODE REVIEW

File: src/pages/top.js [hunk 1]
```diff
@@ -192,26 +209,36 @@ function Top({ trees, planters, countries, organizations }) {
...
+        const data = await utils.requestAPI('/organizations/featured');
+        log.warn('organizations', data);
+        return data.organizations;
+      })(),
+      (async () => {
+        const data = await utils.requestAPI(
+          '/tokens?wallet=ba95a148-7ff8-42bb-8028-99b747b86ae1',
+        );
+        log.warn('wallets', data);
+        return data.tokens;
```
Comment (dadiorchen) [inline src/pages/top.js:231]: This is wrong, this return tokens, we need wallets (the top page will show wallets picked by us in advance, so called `featured wallet`)

So you should propobly go: `https://dev-k8s.treetracker.org/query/wallets?limit=10`

I checked, seems this API is missing, I opened an issue for this:

https://github.com/Greenstand/treetracker-query-api/issues/146

**Comment by dadiorchen** [comment]
@jeremydthomas and please not, one integration test is failing (check error above) let me know you don't know how to fix it

File: src/pages/top.js (hunk 1 above)
Comment (jeremydthomas) [reply src/pages/top.js:231]: I figured it was wrong, but it's the only thing I could find. How do I see a list of all the apis available? I couldn't figure that out.

**Comment by dadiorchen** [comment]
@jeremydthomas this one is perfect

**Pull request merged**